src/main/src/main/java/org/geoserver/config/impl/GeoServerInfoImpl.java

```diff
@@ -50,6 +50,8 @@ public class GeoServerInfoImpl implements GeoServerInfo {
 
     protected Boolean xmlExternalEntitiesEnabled = Boolean.FALSE;
 
+    protected Boolean trailingSlashMatch = Boolean.FALSE;
```

Comment:
Should be true by default, we might decide to set it to false down the line, in preparation for a Spring 6 upgrade (where doing a trailing slash match will require more than just configuration)

Reply:
This could be controlled by a system variable, e.g. ``Boolean.getValue(System.getProperty("myProperty", "true))``

Reply:
changed